.github/workflows/build.yml

```diff
@@ -96,16 +95,24 @@ jobs:
       previewctl_hash: ${{ steps.build.outputs.previewctl_hash }}
     steps:
       - uses: actions/checkout@v3
+      - name: Configure workspace
+        run: |
+          # Needed by docker/login-action
+          sudo chmod goa+rw /var/run/docker.sock
```

Comment:
What error do you see without this?

Reply:
https://github.com/gitpod-io/gitpod/actions/runs/5519828597/jobs/10065651199#step:4:58

But this is just an attempt, no green build yet.